packages/cli/package.json

```diff
@@ -3,11 +3,18 @@
     "version": "0.59.7",
     "description": "Nango's CLI tool.",
     "type": "module",
-    "main": "dist/index.js",
     "bin": {
-        "nango": "dist/index.js"
+        "nango": "./dist/index.js"
+    },
+    "main": "./dist/sdkScripts.js",
+    "types": "./dist/sdkScripts.d.ts",
+    "typings": "./dist/sdkScripts.d.ts",
+    "exports": {
+        ".": {
+            "require": "./dist/sdkScripts.js",
+            "types": "./dist/sdkScripts.d.ts"
+        }
```

Comment:
We re-expose the types from runner-sdk so that we don't need users to install an other packages.